package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -165,3 +176,212 @@ def _read_first_frame(self):
         self.ts.frame = 0
         if self.convert_units:
             self.convert_pos_from_native(self.ts._pos)  # in-place !
+            try:
+                self.convert_velocities_from_native(self.ts._velocities)  # in-place !
+            except AttributeError:
+                pass
+
+class DATAWriter(base.Writer):
+    """Write out the current time step as a LAMMPS DATA file.
+
+    This writer supports the sections Atoms, Masses, Velocities, Bonds,
+    Angles, Dihedrals, and Impropers. This writer will write the header
+    and these sections (if applicable). Atoms section is written in the
+    "full" sub-style if charges are available or "molecular" sub-style
+    if they are not. Molecule id is set to 0 for all atoms.
+
+    .. Note::
```

Comment:
http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_numpy.html

We're trying to move doc strings to this style instead, it looks better in ipython and the source